geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -1,6 +1,16 @@
 This file includes migration steps for each release of c2cgeoportal.
 
 
+Version 2.6.0
+=============
+
+Changes to apply
+----------------
+
+1. All the ngeo components will now be in the constants thats defined in the vars files in interfaces_config.
+   Theres is some new documentation in ngeo about that.
```

Comment:
Can we be more specific? Is it what is in the c2cgeoportal documentation doc/integrator/ngeo.rst.tmpl, or is something from the ngeo project documentation?

Reply:
Yes, in this pull request...

Reply:
So, instead of "Theres is some new documentation in ngeo about that." I would propose:
"Please refer to the Integrator Guide for more information."

Reply:
Oups => Updated :-)